package.json

```diff
@@ -29,6 +29,7 @@
     "test:lockfile": "lockfile-lint -p package-lock.json -t npm -a npm -o https: -c -i",
     "test:lint": "eslint --ext js,jsx,md,snap .",
     "start": "node lib/server/index.js",
+    "start:inspect": "node --inspect --expose-gc lib/server/index.js",
     "test:unit": "jest --testPathIgnorePatterns integration --config jest.config.js",
```

Comment:
No convenient npm script for this functionality?

Reply:
this gives us that `npm run start:inspect` ?

Reply:
`npm run serve:prod-sample-module ../some-test-module`?